boilerplate/app/app.tsx

```diff
@@ -12,17 +12,29 @@
 import "./i18n"
 import "./utils/ignore-warnings"
 import { useFonts } from "expo-font"
-import React, { useEffect, useState } from "react"
+import React from "react"
 import { initialWindowMetrics, SafeAreaProvider } from "react-native-safe-area-context"
-import { RootStore, RootStoreProvider, setupRootStore } from "./models"
+import { useInitialRootStore } from "./models"
```

Comment:
This is a new custom hook that moves the root store setup out of the app.tsx. I'm open to other naming ideas here. `useInitialRootStore` is okay, but I'm not sure if it really captures what's happening.